pkg/configprofiles/testdata/virtual-app

```diff
@@ -27,7 +27,6 @@ WHERE variable IN (
 ORDER BY variable
 ----
 kv.rangefeed.enabled false
-physical_replication.enabled false
```

Comment:
After some time trying to understand why this happened, I sort of gave up 
and simply resorted to `--rewrite`, since this is a test-only change. 
I guess it has something to do with `ccl.TestingEnableEnterprise()()`.

Reply:
Yeah, I think if you want to rbing this back you'd perhaps need to add an empty import into the test file.

Reply:
Noted. But I think it's not adding any value so I'll skip that.